test/utils.cpp

```diff
@@ -12,8 +12,7 @@ TEST_CASE("tokenize() extracts tokens separated by given delimiters", "[utils]")
 	std::vector<std::string> tokens;
 
 	SECTION("Default delimiters")
-	{
-		tokens = utils::tokenize("as df qqq");
+	{ tokens = utils::tokenize("as df qqq");
```

Comment:
Looks like you accidentally broke the formatting here.

Reply:
Whoops